uv/reporter/store.py

```diff
@@ -54,8 +54,12 @@ class LambdaReporter(AbstractReporter):
 
   Args:
     report: Function called whenever reporter.report(step, k, v) is called.
-    report_all: Function called whenever reporter.reporter_all(step, m) is
+    report_all: Function called whenever reporter.report_all(step, m) is
```

Comment:
I modified the bases to handle the new params reporting too. `fs` can't do it yet, nor can tensorboard... but that actually probably COULD support it.